collector/benchmarks/style-2f3bc0de49/makefile

```diff
@@ -19,20 +19,18 @@ all@020-clean:
 	    $(CARGO_RUSTC_OPTS) -Z incremental-info
 
 all@030-commit1:
-	patch -Np1 -i 030-commit-b5762295.patch
+	patch -Np1 -i 030-debugging-println.patch
 	touch components/style/lib.rs
 	CARGO_INCREMENTAL=1 \
 	    $(CARGO) rustc $(CARGO_OPTS) -- \
 	    $(CARGO_RUSTC_OPTS) -Z incremental-info
```

Comment:
Would you mind removing `-Z incremental-info` here and everywhere else in this file? It might skew the timings since it causes the compiler to do more work than it has to.

Reply:
Done.